src/Xamarin.Android.Sdk/targets/Xamarin.Android.Sdk.targets

```diff
@@ -7,11 +7,55 @@
 
   <PropertyGroup>
     <XamarinAndroidSdkTargetsImported>true</XamarinAndroidSdkTargetsImported>
+    <_XamarinAndroidBuildTasksAssembly>$(MSBuildThisFileDirectory)..\tools\Xamarin.Android.Build.Tasks.dll</_XamarinAndroidBuildTasksAssembly>
   </PropertyGroup>
 
   <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.targets" />
 
   <Import Project="$(MSBuildThisFileDirectory)..\tools\Xamarin.Android.DefaultOutputPaths.targets" Condition="'$(EnableDefaultOutputPaths)' == 'true'" />
   <Import Project="$(MSBuildThisFileDirectory)..\tools\Xamarin.Android.Common.targets" />
 
+  <UsingTask TaskName="Xamarin.Android.Tasks.ValidateJavaVersion" AssemblyFile="$(_XamarinAndroidBuildTasksAssembly)" />
+  <UsingTask TaskName="Xamarin.Android.Tasks.ResolveAndroidTooling" AssemblyFile="$(_XamarinAndroidBuildTasksAssembly)" />
+
```

Comment:
Should we put the additional `<UsingTask/>` and target in a new `.targets` file? Maybe `Xamarin.Android.Sdk.Common.targets`?

Reply:
I would put the UsingTask in the target which is using them.

Reply:
I think this makes sense for now, `Xamarin.Android.Sdk.targets` can be treated as our "Common" file, and we can create additional targets as necessarily. 

Depending on how far we want to split things up at this time however, we could consider putting these in a new `Xamarin.Android.Sdk.ValidateTooling.targets` file or something?

Reply:
yes something like `Xamarin.Android.ValidateToolng.targets` or `Xamarin.Android.ResolveTools.targets`. That is was I was envisioned, so its all kinda self contained.

Reply:
I put the targets in `Xamarin.Android.Tooling.targets`, which binding projects and regular projects share now.

It seems like all project types _build_ now, but let's see what breaks on CI.